packages/web-frontend/src/containers/Map/DataVisualsLayer/DataVisualsLayer.js

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
Thanks for doing this refactor, so much cleaner now! Great to break up big components into smaller ones.

Minor: I can guess at what the refactor is from the code change, but would be beneficial to name the commit something like "Refactor: extract Map marker and polygon rendering out into separate component", makes it easier to review. No need to change it this time though!

Reply:
Good point. Can do better naming on the commit.